### ISSUE TITLE
Calling execute on fake backend with pulse schedule gives error

### ISSUE THREAD

**Issue by taalexander** [Qiskit/qiskit-aer#732]
### Informations

- **Qiskit Aer version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?
An error is raised if I try and call the code below.

### Steps to reproduce the problem
```python
from qiskit import execute
from qiskit import pulse
from qiskit.test.mock import FakeOpenPulse2Q

backend = FakeOpenPulse2Q()

d0 = pulse.DriveChannel(0)
sched = pulse.Schedule()
sched +=pulse.Play(pulse.Constant(100, 0.5), d0)
execute(sched, backend)
```
yields
```
     10     pulse.play(pulse.Constant(100, 0.5), d0)
     11 pulse_prog.draw()
---> 12 execute(pulse_prog, backend)

~/anaconda3/envs/qiskitv0.18/lib/python3.8/site-packages/qiskit/execute.py in execute(experiments, backend, basis_gates, coupling_map, backend_properties, initial_layout, seed_transpiler, optimization_level, pass_manager, qobj_id, qobj_header, shots, memory, max_credits, seed_simulator, default_qubit_los, default_meas_los, schedule_los, meas_level, meas_return, memory_slots, memory_slot_size, rep_time, parameter_binds, schedule_circuit, inst_map, meas_map, scheduling_method, **run_config)
    281     # executing the circuits on the backend and returning the job
    282     start_time = time()
--> 283     job = backend.run(qobj, **run_config)
    284     end_time = time()
    285     _log_submission_time(start_time, end_time)

~/anaconda3/envs/qiskitv0.18/lib/python3.8/site-packages/qiskit/test/mock/fake_backend.py in run(self, qobj)
    119             if qobj.type == 'PULSE':
    120                 from qiskit.providers.aer.pulse import PulseSystemModel
--> 121                 system_model = PulseSystemModel.from_backend(self)
    122                 sim = Aer.get_backend('pulse_simulator')
    123                 job = sim.run(qobj, system_model)

~/anaconda3/envs/qiskitv0.18/lib/python3.8/site-packages/qiskit/providers/aer/pulse/pulse_system_model.py in from_backend(cls, backend, subsystem_list)
    157                         if len(u_string) > 0:
    158                             u_string += ' + '
--> 159                         u_string += str(scale[0] + scale[1] * 1j) + 'q' + str(q_idx)
    160                     control_channel_labels[u_idx] = {'driven_q': drive_idx, 'freq': u_string}
    161 

TypeError: 'complex' object is not subscriptable
```

### What is the expected behavior?
Either a user-facing error should be raised or this should do something (simulate a pulse system?)

**Comment by mtreinish** [comment]
This should have been fixed in aer 0.15.1 (qiskit-metapackage 0.18.3) See https://github.com/Qiskit/qiskit-aer/pull/695  for the fix

**Comment by taalexander** [comment]
Thanks.
